### ISSUE TITLE
[flutter_svg] Allow animation/transition between placeholderBuilder and the SVG

### ISSUE THREAD

**Issue by stuartmorgan-g** [flutter/flutter#158715]
_Imported from https://github.com/dnfield/flutter_svg/issues/796_

Original report by @lille-morille on Oct 22, 2022

Having some sort of animation control for fading etc between the placeholder and SVG would be nice.
Flutter is great for fluid animations and transitions, and simply snapping between two images looks off in many apps.
Following the Material Design Guidelines, loading images should fade between the loading and complete state. [[LINK]](https://m1.material.io/patterns/loading-images.html#)

Please consider my suggestion. I love this project, thank you for a great package! <3

---
Comment by @behnamsattar on Jan 25, 2023
Or maybe even consider adding a [frameBuilder](https://api.flutter.dev/flutter/widgets/Image/frameBuilder.html) property the way it's used for the built-in Image widget.

---
Comment by @jaggernod on Mar 9, 2023
Before v2.0 it was relatively simple to have the crossfade. However, required changing the top level of `SvgPicture` (not the cleanest solution).

In v2.+ `flutter_svg` does not have enough information to handle the animation. 
Now [vector_graphics](https://github.com/dnfield/vector_graphics/issues/193) would need to expose the feature.

**Comment by darshankawar** [comment]
> Now [vector_graphics](https://github.com/dnfield/vector_graphics/issues/193) would need to expose the feature.

This was added recently as part of https://github.com/flutter/packages/pull/8195, @lille-morille does this help to resolve your case ?

**Comment by lille-morille** [comment]
Yes! Although I stopped using the library, this seems to resolve the issue. Thanks for fixing this. We can mark as closed.

**Comment by darshankawar** [comment]
Good to know. Closing as fixed.
